apps/condo/domains/marketplace/schema/MarketSetting.test.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * Generated by `createschema marketplace.MarketSetting 'organization:Relationship:Organization:CASCADE; residentAllowedPaymentTypes:Select:cash, online;'`
+ */
+
+const { faker } = require('@faker-js/faker')
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE, waitFor, expectValuesOfCommonFields, catchErrorFrom } = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj, expectToThrowAccessDeniedErrorToObjects,
+} = require('@open-condo/keystone/test.utils')
+
+const { INVOICE_PAYMENT_TYPES } = require('@condo/domains/marketplace/constants')
+const { MarketSetting, createTestMarketSetting, updateTestMarketSetting } = require('@condo/domains/marketplace/utils/testSchema')
+const { createTestOrganization } = require('@condo/domains/organization/utils/testSchema')
+const {
+    createTestOrganizationEmployeeRole,
+    createTestOrganizationEmployee,
+} = require('@condo/domains/organization/utils/testSchema')
+const { createTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { createTestResident, createTestServiceConsumer } = require('@condo/domains/resident/utils/testSchema')
+const { makeClientWithResidentUser } = require('@condo/domains/user/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
+
+describe('MarketSetting', () => {
+    let admin, support, anonymous, user
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
+        support = await makeClientWithSupportUser()
+        user = await makeClientWithNewRegisteredAndLoggedInUser()
+        anonymous = await makeClient()
+    })
+    describe('access tests', () => {
+        describe('admin', () => {
+            test('can create', async () => {
+                const [organization] = await createTestOrganization(admin)
+                const [obj, attrs] = await createTestMarketSetting(admin, organization)
+
+                expectValuesOfCommonFields(obj, attrs, admin)
```

Comment:
how about checking the other fields? (`organization`, `residentAllowedPaymentTypes`)